Crons initial docs (#5985)
getsentry/sentry-docs: src/docs/product/crons/job-monitoring.mdx

---
title: Job Monitoring
sidebar_order: 1
---

> Sentry's Cron Monitoring service is able to notify you and store a timeline of the following check-ins events:

Reviewer: ```suggestion
Sentry's Cron Monitoring service can notify you and store a timeline of the following check-in events:
```

> - Missed check-in. The job did not execute in the predicted timeframe or frequency, and this can happen for various reasons, such as:
>   - The job scheduler is misconfigured, skipped, or failed to initiate your job.
>   - A timeout failure event when a job sent an initial check-in but failed to send a final check-in.
>   - Network issues, such as an outbound firewall or an unstable connection.
>   - An invalid request format.
> - Failed check-in. The job has reported its execution as unsuccessful.
> - Successful check-in. The job has reported its execution as successful.

Reviewer: ```suggestion
- **Missed check-in:** The job didn't execute in the predicted timeframe or frequency. This can happen for various reasons, such as:
  - The job scheduler is misconfigured, skipped, or failed to initiate your job.
  - A timeout failure event when a job sent an initial check-in, but failed to send a final check-in.
  - Network issues, such as an outbound firewall or an unstable connection.
  - An invalid request format.
- **Failed check-in:** The job has reported its execution as unsuccessful.
- **Successful check-in:** The job has reported its execution as successful.
```

> To see information about your cron monitor, go to "Crons" in the sidebar menu and click on your Cron Monitor's name to see more details.

Reviewer: ```suggestion
To see information about your cron monitor, select "Crons" from the sidebar menu and click on your cron monitor's name.
```

> ## Cron monitor details

Reviewer: ```suggestion
## Cron Monitor Details
```

> On your Cron monitor details page, you will find a daily historical bar chart for successful, failed, and missed check-ins with a line chart of the runtime average for the job execution.

Reviewer: ```suggestion
This is where you'll find a daily historical bar chart showing successful, failed, and missed check-ins and a line chart of the runtime average for the job execution.
```

> Below, in the "Issues" section, any issues created from this Cron Monitor will be listed here. Issues are created when a cron monitor job execution is missed or failed. If you have configured the Sentry SDK for your job, any errors thrown during the job runtime will be shown here as well.
> 
> The table of recent check-ins lists previously scheduled jobs and their status.

Reviewer: ```suggestion
The "Issues" section shows the issues created from this Cron Monitor. Issues are created when a cron monitor job execution is missed or failed. If you have configured the Sentry SDK for your job, any errors thrown during the job runtime will be shown here as well.

The table of recent check-ins lists previously scheduled jobs and their status.
```

> ## Pausing your cron monitor

Reviewer: ```suggestion
## Pausing Your Cron Monitor
```

> In the header of your cron monitor, you are able to pause your job monitoring. By pausing, your monitor will not record any check-ins or notify you of any failures or missed check-ins.
> 
> Note: pausing your cron monitor does not pause the execution of your job nor does it change the configuration of your job scheduler.

Reviewer: ```suggestion
You can pause job monitoring in the header of your cron monitor. When you pause, your monitor will stop recording check-ins and will no longer notify you of any failed or missed check-ins.

Note: pausing your cron monitor won't pause the execution of your job or change the configuration of your job scheduler.
```
